runtime/moonriver/src/lib.rs

```diff
@@ -961,6 +988,23 @@ parameter_types! {
 			)
 		)
 	};
+	// Bew Self Reserve location, defines the multilocation identifiying the self-reserve currency
+	// This is used to match it also against our Balances pallet when we receive such
+	// a MultiLocation: (Self Balances pallet index)
+	// This is the new anchoring way
+	pub NewAnchoringSelfReserve: MultiLocation = MultiLocation {
+		parents:0,
+		interior: Junctions::X1(
+			PalletInstance(<Runtime as frame_system::Config>::PalletInfo::index::<Balances>().unwrap() as u8)
```

Comment:
Please use unwrap_or_default instead of unwrap, or eventually expect.

We need to remove all unwrap() from production code (except tests)

Reply:
perfect